refs/biblio.json

```diff
@@ -953,10 +953,98 @@
         "href": "https://www.itu.int/rec/dologin_pub.asp?lang=e&id=T-REC-E.164-201011-I!!PDF-E&type=items",
         "date": "November 2010"
     },
+    "EBU-TT": {
+      "title": "EBU TECH 3350: \"EBU-TT Subtitling format definition\"",
+      "href": "https://tech.ebu.ch/docs/tech/tech3350.pdf",
+      "publisher": "European Broadcasting Union",
+      "versions": {
+        "1": {
+          "aliasOf": "EBU-TT-1-0"
+        },
+        "1.1": {
+          "aliasOf": "EBU-TT-1-1"
+        },
+        "1.2": {
+          "aliasOf": "EBU-TT-1-2"
+        }
+      }
+    },
+    "EBU-TT-1-0": {
+        "title": "EBU TECH 3350: \"EBU-TT Subtitling format definition\", Version 1.0",
+        "href": "https://tech.ebu.ch/docs/tech/tech3350v1-0.pdf",
+        "publisher": "European Broadcasting Union",
+        "date": "July 2012"
+    },
+    "EBU-TT-1-1": {
+        "title": "EBU TECH 3350: \"EBU-TT Subtitling format definition\", Version 1.1",
+        "href": "https://tech.ebu.ch/docs/tech/tech3350v1_1.pdf",
+        "publisher": "European Broadcasting Union",
+        "date": "September 2015"
+    },
+    "EBU-TT-1-2": {
+        "title": "EBU TECH 3350: \"EBU-TT Subtitling format definition\", Version 1.2",
+        "href": "https://tech.ebu.ch/docs/tech/tech3350v1-2.pdf",
+        "publisher": "European Broadcasting Union",
+        "date": "May 2017"
+    },
+    "EBU-TT-STL": {
+        "title": "EBU TECH 3360: \"EBU-TT, Part 2 Mapping EBU STL (TECH 3264) to EBU-TT Subtitle Files\"",
+        "href": "https://tech.ebu.ch/docs/tech/tech3360.pdf",
+        "publisher": "European Broadcasting Union",
+        "date": "May 2017"
+    },
+    "EBU-TT-Live": {
+      "title": "EBU TECH 3370: \"EBU-TT, Part 3 Live Subtitling Applications\"",
+      "href": "https://tech.ebu.ch/docs/tech/tech3370.pdf",
+      "publisher": "European Broadcasting Union",
+      "versions": {
+        "1": {
+          "aliasOf": "EBU-TT-Live-1-0"
```

Comment:
Same question here, what's the purpose of having these different ways to refer to these specs?